#Request is an integer called “start”
int32 start
---
#Response is a boolean called “confirm”
bool confirm